vscode/src/cli.ts

```diff
@@ -9,7 +9,12 @@ export function toitExecFileSync(ctx: Context, ...args: string[]): string {
 }
 
 export function toitExecFilePromise(ctx: Context, ...args: string[]): Promise<{stdout: string, stderr: string}> {
-  return execFile(ctx.toitExec, args);
+  let exec = ctx.toitExec;
+  if (args.length > 0 && args[0] === "pkg") {
+    exec = ctx.pkgExec;
+    args.push("--cache", "~/.cache/toit/tpk/tmp/", "--config", "~/.config/toit/toit.yaml");
```

Comment:
You probably tested it, but just in case: the `~` are usually only expanded by shells. If `execFile` runs in a shell that should happen. Otherwise it won't work and you need to replace `~` with the `HOME` env variable yourself.

Reply:
I didn't test it, but I encountered this problem when I started using it.